interpreter/value_dictionary.go

```diff
@@ -730,21 +730,6 @@ func (v *DictionaryValue) MeteredString(context ValueStringContext, seenReferenc
 }
 
 func (v *DictionaryValue) GetMember(context MemberAccessibleContext, locationRange LocationRange, name string) Value {
-	if context.TracingEnabled() {
-		startTime := time.Now()
-
-		typeInfo := v.Type.String()
-		count := v.Count()
-
-		defer func() {
-			context.ReportDictionaryValueGetMemberTrace(
-				typeInfo,
-				count,
-				name,
-				time.Since(startTime),
-			)
-		}()
-	}
```

Comment:
Is this no longer needed?

Reply:
I removed it for now because it was inconsistent with the other containers traces, and didn't seem very useful (e.g. dictionaries have few member methods, and it's not very interesting if e.g. `keys` or `values` is accessed).  We can add it back for all at some point if needed.